canonical-web-and-design/maas-ui
# Site Identity - MAAS Theme Colours

## Done

- Created custom radio buttons to select theme colours
- Apply theme colours to site header
- Wrote additional tests for themes

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps
- Verify that theme colours are correct
- Verify that theme can be changed

## Screenshots

![image](https://user-images.githubusercontent.com/35104482/178515446-d56d00f0-77a8-4d91-b1f4-60c944d132b7.png)

It could be helpful to provide some screenshots to aid in QAing the change.
